.github/workflows/progress-tracker.yml

```diff
@@ -139,7 +142,7 @@ jobs:
             }
 
             const formatTeam = team => {
-              return Object.entries(team).map(([k, v]) => `<span title="${k}: ${v.length ? v.join(', ') : 'TBD'}">${v.length}</span>`).join('/')
+              return Object.entries(team).map(([k, v]) => `<span title="${k}: ${v.length ? v.join(', ') : 'TBD'}">${v.length}</span>`).join('+').replace(/\+([^+]*)$/, '=$1')
```

Comment:
How about keeping "A/B/C" format and dealing with the fact that they won't add up to the unique total by formatting it as "A/B/C (D total)"?

Reply:
I have changed it slightly differently to keep it logically correct while still being compact.